### ISSUE TITLE
DMRS representation: sortinfo

### ISSUE THREAD

**Issue by arademaker** [own-pt/delphin-rdf#23]
Related to #21. 

For many tasks, we may not need the variable properties. This are the extra (some morphological) information attached to variables in MRS and projected to notes in DMRS ([example](http://delph-in.github.io/delphin-viz/demo/#input=A%20man%20is%20crying&count=5&grammar=erg2018-uw&mrs=true&dmrs=true)).  

In the DMRS XML DTD (see https://github.com/delph-in/docs/wiki/RmrsDmrs) these are grouped into a tag descendent from the node tag. I believe we should do the same for RDF. 

```
<sortinfo SF="prop" TENSE="untensed" MOOD="indicative" PROG="bool" PERF="-" cvarsort="e" />
```

So, instead of

```
http://example.com/a/nodes/10003 http://www.delph-in.net/schema/erg#perf -
http://example.com/a/nodes/10003 http://www.delph-in.net/schema/erg#mood indicative
http://example.com/a/nodes/10003 http://www.delph-in.net/schema/erg#prog +
http://example.com/a/nodes/10003 http://www.delph-in.net/schema/erg#sf prop
```

We can have (note URIs according to https://github.com/own-pt/delphin-rdf/issues/21#issuecomment-871773857). The URI for the new node is because a sortinfo is always attached to a Node.

```
http://example.com/a/1#node-10003 dmrs:sortInfo http://example.com/a/1#sortinfo-10003


http://example.com/a/1#sortinfo-10003 rdf:type dmrs:SortInfo
http://example.com/a/1#sortinfo-10003 http://www.delph-in.net/schema/erg#perf "-"
http://example.com/a/1#sortinfo-10003 http://www.delph-in.net/schema/erg#mood "indicative"
http://example.com/a/1#sortinfo-10003 http://www.delph-in.net/schema/erg#prog "+"
http://example.com/a/1#sortinfo-10003 http://www.delph-in.net/schema/erg#sf "prop"
```

**Comment by arademaker** [comment]
This was solved by cc27e6b

**Comment by arademaker** [comment]
OK, let us see what will be the comments in https://github.com/delph-in/docs/pull/27
